### DISCUSSION TITLE
These diagrams seems all the same,What is the point of this example?

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#167] [Q&A]
**Below is the example code of function rfsrc :**
library(randomForestSRC)
iris.obj <- rfsrc(Species ~., data = iris)
plot(get.tree(iris.obj, 25, class.type = "bayes"))
plot(get.tree(iris.obj, 25, target = "setosa"))
plot(get.tree(iris.obj, 25, target = "versicolor"))
plot(get.tree(iris.obj, 25, target = "virginica"))
![image](https://user-images.githubusercontent.com/5112397/149259110-b74bd511-791e-43d0-8f16-af2df61e62a6.png)
![image](https://user-images.githubusercontent.com/5112397/149259723-e85616eb-5bc3-4fa6-a7cf-2bf9b5e0b4de.png)
![image](https://user-images.githubusercontent.com/5112397/149259365-94275089-cfd9-4f48-83c0-56461cf7597e.png)
![image](https://user-images.githubusercontent.com/5112397/149259396-a5bf05b6-cf71-4752-96d1-67ca493c43b1.png)

**These diagrams seems all the same,What is the point of this example?**

**Comment by ishwaran** [comment]
Thanks for the comment.  I will improve this example.
